src/corporacreator/preprocessors/fr.py

```diff
@@ -1,3 +1,19 @@
+import re
+
+# All special characters
+FILTER_SYMBOLES_REG = re.compile(
```

Comment:
While `FILTER_SYMBOLES_REG` seems useful what might be better is a white list instead of a black list.

For example, the characters 男性 are not in your black list and would pass through un-altered.